### PR TITLE
AUT-1124 - stopped IPN support page test from submitting to support desk

### PR DESCRIPTION
**Pull request by CraigRyder**
## What?

Removed 'Send Message' step within IPN support pages acc test and refactored accordingly.

## Why?

Tests were submitting cases to the actual support desk.

### CODE REVIEW

File: acceptance-tests/src/test/resources/uk/gov/di/test/features/IPN.feature [hunk 1]
```diff
@@ -83,12 +83,12 @@ Feature: International Phone Numbers
...
     Then the user is taken to the Contact us page in a new tab
-    When the user selects "A problem creating a GOV.UK One Login" and proceeds
+    When the user selects "A problem creating a GOV.UK One Login"
     Then the user is taken to the "A problem creating a GOV.UK One Login" page
-    When the user selects "You did not get a security code" and proceeds
+    When the user selects "You did not get a security code"
     Then the user is taken to the "You did not get a security code" page
-    When the user selects "Text message to a phone number from another country" and proceeds
-    Then the user receives confirmation that their message has been submitted
+    And the user selects "Text message to a phone number from another country"
```
Comment (JHjava) [inline acceptance-tests/src/test/resources/uk/gov/di/test/features/IPN.feature:90]: Is it worth getting rid of this `And` condition too? Does it offer us anything?

File: acceptance-tests/src/test/resources/uk/gov/di/test/features/IPN.feature [hunk 2]
```diff
@@ -107,9 +107,9 @@ Feature: International Phone Numbers
...
-    When the user selects "A problem signing in to your GOV.UK One Login" and proceeds
+    When the user selects "A problem signing in to your GOV.UK One Login"
     Then the user is taken to the "A problem signing in to your GOV.UK One Login" page
-    When the user selects "The security code did not work" and proceeds
+    When the user selects "The security code did not work"
     Then the user is taken to the "The security code does not work" page
-    When the user selects "Text message to a phone number from another country" and proceeds
-    Then the user receives confirmation that their message has been submitted
\ No newline at end of file
+    And the user selects "Text message to a phone number from another country"
```
Comment (JHjava) [inline acceptance-tests/src/test/resources/uk/gov/di/test/features/IPN.feature:114]: Similar to above. Does this `And` offer us anything?

**Review by JHjava — COMMENTED** [review]
Couple of questions

File: acceptance-tests/src/test/resources/uk/gov/di/test/features/IPN.feature (hunk 1 above)
Comment (CraigRyder) [reply acceptance-tests/src/test/resources/uk/gov/di/test/features/IPN.feature:90]: That step is selecting the new radio button that was put in for IPN so I think there is still value in it being there as it's proving that the button exists and can be selected, even if we're no longer going to actually submit the case.

File: acceptance-tests/src/test/resources/uk/gov/di/test/features/IPN.feature (hunk 2 above)
Comment (CraigRyder) [reply acceptance-tests/src/test/resources/uk/gov/di/test/features/IPN.feature:114]: Response same as above.

File: acceptance-tests/src/test/resources/uk/gov/di/test/features/IPN.feature (hunk 1 above)
Comment (JHjava) [reply acceptance-tests/src/test/resources/uk/gov/di/test/features/IPN.feature:90]: Cool. That makes sense.

**Review by JHjava — APPROVED** [review]
👍

**Pull request merged**